kf_lib_data_ingest/factory/templates/my_study/accounting_config.py

```diff
@@ -0,0 +1,49 @@
+"""
+Auto-generated accounting config module
+
+Replace the contents of test_parameters
+
+See documentation at
+https://kids-first.github.io/kf-lib-data-ingest/ for information on
+implementing the accounting config.
+"""
+
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.etl.accounting import test
+
+# **** Define custom test functions here **** #
+
+
+test_parameters = [
+
+    # **** Auto-generated default tests **** #
+    {
+        'function': test.unique_concept_count,
+        'inputs': {
+            'concept': CONCEPT.PARTICIPANT.ID,
+            'expected': 100,  # we expect exactly 100 participants
+            'test': '=='
+        },
+        'enable': True
```

Comment:
Allows an ingest configurator to turn on/off certain tests. Could be good for debugging?

Reply:
Disagree. At least in this example. This is just a more verbose version of commenting out a single line that expresses the expected count for the given key.